my readme file in python exception
